app-deployment-manager/pkg/fleet/configgen.go

```diff
@@ -648,8 +648,21 @@ func BundleName(app v1beta1.Application, depName string) string {
 	}
 
 	hashInHex := hex.EncodeToString(hash.Sum(nil))
+
+	// Generate a UUID using the SHA-1 hash as a namespace
 	u := uuid.NewSHA1(uuid.Nil, []byte(hashInHex))
-	return fmt.Sprintf("b-%s", u.String())
+	uuidStr := u.String()
+	truncatedUUIDLength := 16
```

Comment:
Would it help to make this configurable

Reply:
Not a bad idea in case we need to increase it but  if people don't misuse the behavior that is the only concern.

Reply:
let me think about it and get back to you. We use the bundleName in tests as well so it could be tricky.